index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>myForm</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;300;400&display=swap" rel="stylesheet">
+</head>
+<body>
+    <form action="https://httpbin.org/anything" method="POST">
+        <fieldset>
+            <legend>Your Donation Matters</legend>
+            <div class="text-field">
+                <label class="textfield-container" for="fname">Name:<br>
+                    <input type="name" id="fname" name="fname" value="Name">
+                </label><br>
```

Comment:
Try styling the form with `margin`, `padding`, maybe using `display: block;` or `display: flex;` to avoid non-semantical HTML elements like `br`